### PR TITLE
HPCC-26151 Managed ELK chart update

### PR DESCRIPTION
**Pull request by rpastrana**
- Upgrades target ELK to 7.120 for bug fixes
- Adds app version to chart
- Updates chart ver
- Provides Sample pipeline, and documentation

Signed-off-by: Rodrigo Pastrana <Rodrigo.Pastrana@lexisnexisrisk.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [x] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by rpastrana** [comment]
@richardkchapman / @jakesmith this is the first step towards full control of the Elastic Search index used to house HPCC log entries. Subsequent enhancements will attempt to automatically deploy HPCC Specific Index pattern (including the pipeline from this PR).

**Review by richardkchapman — APPROVED** [review]

File: helm/managed/logging/elastic/README.md [hunk 1]
```diff
@@ -1,19 +1,109 @@
-## This folder contains lightweigth Elastic Stack deployment charts and HPCC prefered values
+## This folder contains light-weight Elastic Stack deployment chart and HPCC Systems preferred values
```
Comment (jakesmith) [inline helm/managed/logging/elastic/README.md:1]: trivial: lightweight normally not hyphenated?

File: helm/managed/logging/elastic/Chart.yaml [hunk 2]
```diff
@@ -5,20 +5,23 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.1.0
+version: 1.0.0
+
+# Elastic Stack version
+appVersion: 7.12.0
```
Comment (jakesmith) [inline helm/managed/logging/elastic/Chart.yaml:11]: it probably doesn't matter, but where does 7.12.0 come from? should it be 8.2.2 
@richardkchapman - I'm not sure if the go*.sh scripts update version in examples - or if they should?

File: helm/managed/logging/elastic/README.md [hunk 3]
```diff
@@ -1,19 +1,109 @@
...
+> helm dependency update <HPCC-Systems Git clone location>/helm/managed/logging/elastic/
 
 ##### HELM Install parameter
 Otherwise, provide the "--dependency-update" argument in the helm install command
 For example:
-> helm install myelastic ./ --dependency-update
+> helm install myelastic <HPCC-Systems Git clone location>/helm/managed/logging/elastic/ --dependency-update
+
+##### Log Query Improvements
+User are encouraged to make use of Elsatic Search "Ingest Pipelines" to improve log query performance and tailor the structure of the log record to meet their needs. The pipelines can be applied to specific indices via Elast Search API, or via Kibana's UI.
```
Comment (jakesmith) [inline helm/managed/logging/elastic/README.md:25]: trivial: Elast -> Elastic

**Review by jakesmith — APPROVED** [review]
Looks fine/approved. Some trivial comments only.

File: helm/managed/logging/elastic/Chart.yaml (hunk 2 above)
Comment (richardkchapman) [reply helm/managed/logging/elastic/Chart.yaml:11]: I think 7.12 is the elk version rather than the HPCC version?

Comment (richardkchapman) [reply helm/managed/logging/elastic/Chart.yaml:11]: And for that reason I'm not sure that the go*.sh scripts should change anything in managed subdir.

Comment (rpastrana) [reply helm/managed/logging/elastic/Chart.yaml:11]: Most of the charts published to our repo via the go. scripts report various chart/app versions already:
 helm search repo hpcc
NAME                    CHART VERSION   APP VERSION     DESCRIPTION
hpcc/elastic4hpcclogs   0.1.0                           A Helm chart for launching a lightweight Elasti...
hpcc/hpcc               8.0.24          8.0.24          A Helm chart for launching a HPCC cluster using...
hpcc/hpcc-azurefile     0.1.0           1.16.0          A helm chart to provision HPCC PVC's using the ...
hpcc/hpcc-efs           0.1.0           1.16.0          A helm chart to provision HPCC PVC's using the ...
hpcc/hpcc-filestore     0.1.0           0.1.0           A helm chart to provision HPCC PVC's using the ...
hpcc/hpcc-localfile     0.1.0           1.16.0          A helm chart to provision HPCC PVC's using the ...
hpcc/hpcc-localplanes   0.1.0           1.16.0          A helm chart to provision multiple HPCC PVC's u...
hpcc/hpcc-nfs           0.1.0           1.16.0          A Helm chart to provision HPCC PVC's using an N...
@richardkchapman

**Comment by rpastrana** [comment]
@jakesmith resolved minor issues.

**Pull request merged**